exersice1.js

```diff
@@ -0,0 +1,100 @@
+// You are talking to a friend who suddently looks at you 
+// and asks: "How old will you be in 2045?" Hmm you remember 
+// the year you were born and try to do some calculation:
+// Born in 1987 + 3 that's 1990. 90 - 40 is 50 + 5 thats 58. 
+// I will be 58! Wow that was painful! Let's fix that by making 
+// some code that automatically does this!
+
+let yearOfBirth = 1986;
+let yearFuture = 2051;
+let age;
+
+age = yearFuture - yearOfBirth;
```

Comment:
This is an unnecessary step. :) You can assign the value as you declare the variable, like so:
 `
const age = yearFuture - yearOfBirth;
`